Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG ALPINE_IMAGE
+ARG ALPINE_IMAGE=alpine:3.18.3
```

Comment:
why is it hardcoded?

Reply:
this is to add default in case it is not provided, it is easier to build locally to test